XamarinCommunityToolkit.UnitTests/Converters/ItemSelectedEventArgsConverter_Tests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using XamarinCommunityToolkit.Converters;
+using System.Collections.Generic;
+using System.Globalization;
+using Xamarin.Forms;
+using System;
+using Xunit;
+
+namespace XamarinCommunityToolkit.UnitTests.Converters
+{
+    public class ItemSelectedEventArgsConverter_Tests
+    {
+        public static object expectedValue = 100;
+
+        public static IEnumerable<object[]> GetData() => new List<object[]>
+        {
+            new object[] { new SelectedItemChangedEventArgs(expectedValue), expectedValue},
+        };
+
+        [Theory]
+        [MemberData(nameof(GetData))]
+        public void InverterBoolConverter(SelectedItemChangedEventArgs value, object expectedResult)
```

Comment:
This is not a correct method name? Too much copy/paste? ;)

Reply:
I'm guilty 😂